.travis.yml

```diff
@@ -25,6 +25,22 @@ matrix:
       install:
       - go list -f '{{range .Imports}}{{.}} {{end}}' ./... | xargs go get -v
       - go list -f '{{range .TestImports}}{{.}} {{end}}' ./... | xargs go get -v
+    - go: 1.7
```

Comment:
You should use a build matrix here - https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix

Example: https://github.com/gorilla/mux/blob/master/.travis.yml

```yml
language: go
sudo: false

matrix:
  include:
    - go: 1.5
    - go: 1.6
    - go: 1.7
    - go: 1.8
    - go: 1.9
    # Important: you must force a string, else YAML will coerce to "1.1", which is NOT the version you want.
    - go: "1.10"
    - go: tip
  allow_failures:
    - go: tip

install:
  - # Skip

script:
  - go get -t -v ./...
  - diff -u <(echo -n) <(gofmt -d .)
  - go tool vet .
  - go test -v -race ./...
```

Reply:
@elithrar thanks! I'll modify you pointed out.